lib/LogsList/LogsList.js

```diff
@@ -5,40 +5,53 @@ import { MultiColumnList, NoValue, Spinner } from '@folio/stripes/components';
 
 const RESULT_COUNT_INCREMENT = 100;
 export default class LogsList extends React.Component {
-  static propTypes = {
-    job: PropTypes.object,
-    logs: PropTypes.arrayOf(PropTypes.shape({
-      recordNumber: PropTypes.string,
-      message: PropTypes.string,
-    })),
-    onNeedMoreLogs: PropTypes.func.isRequired,
-    type: PropTypes.string.isRequired,
-  };
+    static propTypes = {
+        job: PropTypes.object,
+        logs: PropTypes.arrayOf(PropTypes.shape({
+            recordDescriptor: PropTypes.string,
+            message: PropTypes.string,
+        })),
+        onNeedMoreLogs: PropTypes.func.isRequired,
+        type: PropTypes.string.isRequired,
+    };
 
-  render() {
-    const { job, logs, onNeedMoreLogs, type } = this.props;
+    resultsFormatter = {
+        recordDescriptor: ({ additionalInfo }) => {
+            return (
+                <div>
+                    {additionalInfo.packageSource ? <FormattedMessage id="stripes-erm-components.packageSource" tagName="div" values={{ source: additionalInfo.packageSource }} /> : null}
+                    {additionalInfo.packageReference ? <FormattedMessage id="stripes-erm-components.packageReference" tagName="div" values={{ ref: additionalInfo.packageReference }} /> : null}
+                    {additionalInfo.rowNumber ? <FormattedMessage id="stripes-erm-components.rowNumber" tagName="div" values={{ rownumber: additionalInfo.rowNumber }} /> : null}
+                    {additionalInfo.recordNumber ? <FormattedMessage id="stripes-erm-components.recordNumber" tagName="div" values={{ recordnumber: additionalInfo.recordNumber }} /> : null}
+                </div>
```

Comment:
With the above requested change the `additionalInfo.packageSource ` could just be replaced with `packageSource`. Similarly with  `packageReference`, `rowNumber` and `recordNumber`. Its more cleaner that way as we dont need to fetch the property from `additionalInfo` everytime.